### DISCUSSION TITLE
Translate ou language select

### DISCUSSION THREAD

**Discussion by BO105** [mriscoc/Ender3V2S1#579] [General]
Hi !
Yep for the firmware for my Ender 3-S1, easy to install and especially works on the first try.
However, we cannot choose the language, which I would have liked in French.
How can we do
Thank you for your answer.

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/issues/342#issuecomment-1233570983
https://github.com/mriscoc/Ender3V2S1/issues/234

**Reply by BO105** [reply]
Hi
Thank you for your response.

I downloaded your version, changed the language to fr in the right file.
Then compile with Visual Studio and Marlin autobuild, the generated ". bin" file does not install in my Ender 3 S1, despite 4 attempts.
It remains stuck on the main logo, forcing me to put back your original firmware, which works fine.
I’m not a compilation ace either.

Thank you for your help.
